src/store/store.ts

```diff
@@ -1,13 +1,17 @@
-import { configureStore, combineReducers } from '@reduxjs/toolkit'
-import { booksSliceReducer } from '../domain/books/booksSlice'
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import { booksSliceReducer, getBooks } from '../domain/books/booksSlice';
+import { cartSliceReducer } from '../domain/cart/cartSlice';
 
 const combinedReducer = combineReducers({
-  books: booksSliceReducer
+  books: booksSliceReducer,
+  cart: cartSliceReducer
 });
 
 export const store = configureStore({
   reducer: combinedReducer
-})
+});
+
+store.dispatch(getBooks());
```

Comment:
Do not dispatch this action here. We should call it inside BooksPage's useEffect.

Reply:
Fixed